core/Pos/Util/Justify.hs

```diff
@@ -0,0 +1,51 @@
+module Pos.Util.Justify
+       ( leftAlign
+       ) where
+
+import           Universum          hiding ((<>))
+
+import qualified Data.List          as List
+import qualified Data.List.NonEmpty as NonEmpty
+import           Data.Semigroup
```

Comment:
Implicit imports are still prohibited :kot:

Reply:
Oops, missed this one.